spec/schemas.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  Item: {
+    type: "object",
+    properties: {
+      itemId: {
+        type: "string",
+        description: "The unique identifier of the item",
+      },
+      quantity: {
+        type: "integer",
+        description: "The quantity of the item in the cart",
+      },
+    },
+  },
+  Cart: {
+    type: "object",
+    properties: {
+      id: {
+        type: "string",
+        description: "The unique identifier of the cart",
+      },
+      items: {
+        type: "array",
+        description: "An array of items in the cart",
+        items: {
+          $ref: "#/components/schemas/Item",
```

Comment:
ako obratis pozornost, istu stvar smo napravili i ovdje...

s obzirom da se schema "item" koristi na vise mjesta, nju smo izdvojili iznad, i referencirali na ovaj nacin..

referencirali smo je ovdje u definicije Cart objekta, ali i u komentaru na ruti POST /mine/item 

(to sam sad primjetion da mi je pogresno, ispravim sad pa pusham.. trebalo bi bit dobro kad budes gleda